numpy/core/tests/test_umath.py

```diff
@@ -1894,6 +1894,15 @@ def __array_ufunc__(self, ufunc, method, *inputs, **kwargs):
         c = np.add(a, b, out=a)
         assert_(c is a)
         assert_equal(c.info, {'inputs': [0, 1], 'outputs': [0]})
+        # While we're at it, use the same class to check that default output
+        # is never passed on.
+        b = np.sin(a, out=None)
+        assert_('outputs' not in np.sin(a, None).info)
+        assert_('outputs' not in np.sin(a, out=None).info)
+        assert_('outputs' not in np.sin(a, out=(None,)).info)
+        assert_('outputs' not in np.modf(a, None)[0].info)
+        assert_('outputs' not in np.modf(a, None, None)[0].info)
+        assert_('outputs' not in np.modf(a, out=(None, None))[0].info)
```

Comment:
I haven't included that since it is deprecated to give an explicit `out` argument which is not a tuple for `nout=2`:
```
In [2]: np.modf(1., out=None)
/usr/bin/ipython3:1: DeprecationWarning: passing a single array to the 'out' keyword argument of a ufunc with
more than one output will result in an error in the future
```